code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -846,7 +845,7 @@
 
 /obj/item/reagent_containers/food/snacks/popcorn
 	name = "popcorn"
-	desc = "Now let's find some cinema."
+	desc = "A light snack of corn seeds that have been heated in oil untill exploided."
```

Comment:
I suggest changing this to "A light snack of corn seed heated in oil until puffed."